lib/cdo/delete_accounts_helper.rb

```diff
@@ -63,7 +63,7 @@ def delete_project_backed_progress(user)
   # @param [Integer] user_id The user to clean the LevelSource-backed progress of.
   def clean_level_source_backed_progress(user_id)
     @log.puts "Cleaning UserLevel"
-    updated_rows = UserLevel.where(user_id: user_id).update_all(level_source_id: nil)
+    updated_rows = UserLevel.with_deleted.where(user_id: user_id).update_all(level_source_id: nil)
```

Comment:
If the user level is deleted, I'm wondering why it's necessary to still remove the level_source_id